packages/story-editor/src/components/panels/panel/shared/title.js

```diff
@@ -254,7 +254,11 @@ function Title({
         aria-controls={panelContentId}
         isCollapsed={isCollapsed}
       >
-        <IconWrapper>{canCollapse && <Icons.ChevronDownSmall />}</IconWrapper>
+        {canCollapse && (
+          <IconWrapper>
+            <Icons.ChevronDownSmall />
+          </IconWrapper>
+        )}
```

Comment:
if `canCollapse` is `false` then the `IconWrapper` shouldn't render, i don't think we have any cases where this _is_ false - until this fun new feature, so it went unnoticed.  Without this update we get this: 
![canCollapse](https://user-images.githubusercontent.com/10720454/153515617-e5f9e7a4-a06d-4a98-bcc4-49941b189f18.gif)